src/TabViewPanResponder.js

```diff
@@ -8,7 +9,7 @@ type Props = SceneRendererProps & {
 }
 
 const POSITION_THRESHOLD = 120;
-const VELOCITY_THRESHOLD = 0.000001;
+const VELOCITY_THRESHOLD = Platform.OS === 'android' ? 0.0000005 : 0.5; // on Android, velocity is way lower, perhaps due to timestamp being in nanosecond
```

Comment:
There's a open PR which fixes this, we can remove this condition when it goes to stable - https://github.com/facebook/react-native/pull/8199